src/Microsoft.AspNetCore.Razor.VSCode/src/RPC/UpdateProjectRequest.ts

```diff
@@ -0,0 +1,20 @@
+/* --------------------------------------------------------------------------------------------
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT License. See License.txt in the project root for license information.
+* ------------------------------------------------------------------------------------------ */
+
+import { IRazorProjectConfiguration } from '../IRazorProjectConfiguration';
+
+export class UpdateProjectRequest {
```

Comment:
I don't really understand why this class is defined. It seems to be a repeat of the `IRazorProjectConfiguration` type. I can see that the names of the `projectFilePath`/`projectPath` property differ, but is there any other reason for this to exist instead of using `IRazorProjectConfiguration` directly everywhere?

Reply:
Update: No forget that comment, I see that `UpdateProjectRequest` is a small subset of what is on `IRazorProjectConfiguration`.

Arguably it would be clearer if `UpdateProjectRequest` was defined as an interface too, since it doesn't contain any logic and is just a DTO. Not critical though!